package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "gulp docs-generate && webpack-dev-server --inline --hot --content-base src/docs/",
     "docs-upload-gh": "scripts/docs-upload-gh.sh",
     "docs-upload": "gulp docs-upload",
-    "jest": "NODE_ENV=test TZ=UTC jest",
+    "jest": "NODE_ENV=test TZ=UTC jest --runInBand",
```

Comment:
It would be good if there was a way to only run in band on CI mode, and continue to multi-thread when running it locally, since they run faster that way.